Include note on system-assigned identities

dazfuller opened the pull request:
Add a note to make it explicitly clear that system-assigned identities are not supported for this activity

dazfuller committed:
Include note on system-assigned identities

Add a note to make it explicitly clear that system-assigned identities are not supported for this activity

Padmalathas reviewed (commented):
Please change
"Identities must configured as..."
to
"Identities must be configured as..."
Once corrected, please take it forward.

dazfuller committed:
fix grammatical error

dazfuller committed:
fix batch customer managed key link

dazfuller committed:
consistency update

remove "./" from link to keep formatting within document consistent with other links
